v3-docs/v3-migration-docs/breaking-changes/call-x-callAsync.md

```diff
@@ -1,29 +1,25 @@
 # Meteor.call x Meteor.callAsync
 
-
-
 ::: tip
 
-It is recommened to use `Meteor.callAsync` instead of `Meteor.call` because of how our
-async API works. `Meteor.call` is still available but it is not recommended to use it, it
-can lead to unexpected behavior.
+Use `Meteor.callAsync` instead of `Meteor.call` when handling async methods. If you use `Meteor.call` with
+async methods it can lead to unexpected behavior.
 
 Using `Meteor.callAsync` will make your code more predictable and easier to maintain.
 
 :::
 
 Example of how to migrate from `Meteor.call` to `Meteor.callAsync`:
 
-
 ::: code-group
 
-
 ```js [v2-client.jsx]
-import { Meteor } from 'meteor/meteor'
+import { Meteor } from "meteor/meteor";
 
 let data, error;
 
-Meteor.call('getAllData', (err, res) => { // [!code highlight]
+Meteor.call("getAllData", (err, res) => {
+  // [!code highlight]
```

Comment:
```suggestion
Meteor.call("getAllData", (err, res) => {   // [!code highlight]
```

Reply:
@denihs does this make sense? Why is it? @henriquealbert